docs/blog/posts/argocd-keptn-health.md

```diff
@@ -0,0 +1,163 @@
+---
+date: 2024-05-15
+authors: [odubajDT]
+description: >
+  In this blog post we will present how Keptn can show application health in integration with ArgoCD.
+categories:
+  - ArgoCD
+  - Application health
+comments: true
+---
+
+# Application health checks with Keptn using ArgoCD
+
+In this blog post we will present a planned Keptn and
+[ArgoCD](https://argo-cd.readthedocs.io/en/stable/) integration to execute
+advanced application health checks using Keptn, where the application itself is deployed
+by ArgoCD and display the Keptn application health status via ArgoCD UI.
+
+Keptn provides an effective way to perform application health checks using the
+pre- or post-deployment tasks and evaluations.
+Compared to ArgoCD application
+health checks, which are evaluating if the application is successfully deployed
+and the workloads are running on the cluster, they do not show if the microservices
+of a single application are able to communicate and cooperate with each other.
+Keptn pre- and post-deployment tasks and evaluations complement the missing functionality
+by providing a straight-forward way to examine the application ability to perform
+actions for which it was developed to do.
+
+<!-- more -->
+
+## How it's going to work?
+
+The integration of Keptn and ArgoCD performing and displaying full application
+health checks is not fully implemented yet, but
+we can have a brief look how it might work in the future.
```

Comment:
I think we can skip this since it's already stated as very first set of words on this blog post :)